docs/source/roundtrip.rst

```diff
@@ -0,0 +1,120 @@
+.. raw:: html
+
+   <!---
+   # SPDX-FileCopyrightText: 2020 Efabless Corporation
+   #
+   # Licensed under the Apache License, Version 2.0 (the "License");
+   # you may not use this file except in compliance with the License.
+   # You may obtain a copy of the License at
+   #
+   #      http://www.apache.org/licenses/LICENSE-2.0
+   #
+   # Unless required by applicable law or agreed to in writing, software
+   # distributed under the License is distributed on an "AS IS" BASIS,
+   # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   # See the License for the specific language governing permissions and
+   # limitations under the License.
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   -->
+   
+Complete roundtrip for caravel_user_project
+===========================================
+
+1. To start the project you need to first create an empty Git project on Github and make sure your repo is public and includes a README
```

Comment:
might be easier to ask people to fork + clone the https://github.com/efabless/caravel_user_project instead, it would take care of step 2. and 3. for them too.

/cc @jeffdi

Reply:
No, because then people would be unable to create multiple projects. You're only allowed one fork per repo.

Reply:
would it help if https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository was enabled for this repo?